lib/monkey/deps/flb_libco/ppc64.c

```diff
@@ -0,0 +1,279 @@
+/* Base: https://github.com/byuu/higan/tree/master/libco/ppc64v2.c */
```

Comment:
I checked "higan" license and it is GPLv3.
So I think its libco is licensed under GPLv3...

Reply:
Yes, higan is licensed under GPLv3. But libco is released under the ISC license.
Please find the LICENSE file at https://github.com/byuu/higan/blob/master/libco/LICENSE

Reply:
Hi @nokute78, 
I mentioned this link for future reference.  I will remove it if it's not required.
Thanks

Reply:
Hi @ManirajDeivendran.

In my understanding, GPLv3 is copyleft license and the code should be open even if the code is proprietary.

higan is licensed under GPLv3 and its libco is also licensed under GPLv3 because of copyleft.
Then, if the code of "higan's libco" will be merged to fluent-bit, fluent-bit will also be licensed under GPLv3.
If user uses fluent-bit as a library , user should make his/her software open.

Recently, Ruby on Rails linked GPL library and it took the time to remove GPL code.
https://www.theregister.com/2021/03/25/ruby_rails_code/

Reply:
Hi @nokute78,

Thanks for your valuable comments.
So we may not take this patch into fluent-bit because of higan is licensed under GPLv3 and its libco is also licensed under GPLv3. Correct me if i my understanding is wrong.

Thanks.

Reply:
> So we may not take this patch into fluent-bit because of higan is licensed under GPLv3 and its libco is also licensed under GPLv3. 

I think so.
Thank you for contribution, but I can't merge it...